base/linalg/arnoldi.jl

```diff
@@ -363,6 +364,35 @@ iterations derived from [`eigs`](@ref).
 # Example
 
 ```jldoctest
+julia> X = sprand(3, 3, 0.5);
```

Comment:
You can use if it you (visibly) set a seed right before hand but it's strongly discouraged.